lib/ReactViews/Analytics/PolygonParameterEditor.jsx

```diff
@@ -38,7 +38,7 @@ const PolygonParameterEditor = React.createClass({
     },
 
     getValue() {
-        const pointsLongLats = this.props.parameterValues[this.props.parameter.id];
+        const pointsLongLats = this.props.parameterValues[this.props.parameter.id][0];
```

Comment:
I'm just digging through the code to understand what sort of a thing `this.props.parameterValues[key]` is.

It looks like:
`<MyDataTab>` → `<DataPreview previewed={this.props.viewState.userDataPreviewedItem}>` → `<InvokeFunction previewed={this.props.previewed}>` → `<ParameterEditor parameterValues={this._parameterValues}>`
where in the last step, 
```this._parameterValues[parameter.id] = parameter.defaultValue``` (`parameter` is an element from `this.props.previewed.parameters` - [link here](https://github.com/TerriaJS/terriajs/blob/master/lib/ReactViews/Analytics/InvokeFunction.jsx#L65)).

Ie. `this.props.parameterValues[key]` is from `ViewState`'s `userDataPreviewedItem.parameters[x].defaultValue` where `userDataPreviewedItem`.

When I look at `WhyAmISpecialFunction.js` for example, it has a `parameters` array. However none of its elements has a `defaultValue` property...  what am I missing here?

Thanks!